src/diffusers/dependency_versions_table.py

```diff
@@ -17,7 +17,6 @@
     "jaxlib": "jaxlib>=0.1.65,<=0.3.6",
     "modelcards": "modelcards>=0.1.4",
     "numpy": "numpy",
-    "onnxruntime": "onnxruntime",
```

Comment:
We do we remove this? cc @anton-l is this ok?

Reply:
I don't remember removing that manually, could be an accident. I'll put it back for now